doc/sphinx/proof-engine/ltac.rst

```diff
@@ -75,8 +75,19 @@ The constructs in :token:`ltac_expr` are :term:`left associative`.
    | [> @for_each_goal ]
    | @tactic_atom
    tactic_atom ::= @int
+   | @string
    | @qualid
    | ()
+   | @ @ident
+   | & @lident
+   | ' @term
+   | constr : ( @term )
+   | open_constr : ( @term )
+   | ident : ( @lident )
+   | pattern : ( @cpattern )
+   | reference : ( @globref )
+   | ltac1 : ( @ltac1_expr_in_env )
+   | ltac1val : ( @ltac1_expr_in_env )
```

Comment:
My suggestion was that this should all be extracted into a single `ltac2_atom` nonterminal, so that it would be clear when reading the Ltac1 chapter that this is irrelevant, and to document this `ltac2_atom` in the Ltac2 chapter.

Reply:
That makes sense.